builtins/amp-img.js

```diff
@@ -112,7 +113,9 @@ export class AmpImg extends BaseElement {
     if (this.getLayoutWidth() <= 0) {
       return Promise.resolve();
     }
-    const src = this.srcset_.select(this.getLayoutWidth(), this.getDpr()).url;
+    const src = user().assert(this.srcset_,
```

Comment:
We already tested the existence in the `buildCallback`. If we just need this for type convergence, this should be `dev().assert()`